Makefile

```diff
@@ -421,7 +427,11 @@ KUBERNETES_SPLIT_YAML = $(shell pwd)/bin/kubernetes-split-yaml
 kubernetes-split-yaml: ## Download kubernetes-split-yaml locally if necessary.
 	$(call go-get-tool,$(KUBERNETES_SPLIT_YAML),github.com/mogensen/kubernetes-split-yaml@v0.3.0)
 
-# go-get-tool will 'go get' any package $2 and install it to $1.
+SETUP_ENVTEST = $(shell pwd)/bin/setup-envtest
```

Comment:
We should follow what is done in the generated Makefile.  So this becomes `ENVTEST =` and the make target below becomes `envtest`.

Reply:
Yes it is definitely simpler like that. I will do that.